exp-player/addon/components/exp-rating-form/component.js

```diff
@@ -264,19 +264,19 @@ var questions = [
             formatLabel: 'format-label',
             labels: [
                 {
-                    rating: 0,
+                    rating: 'number1',
```

Comment:
I'll merge this as is for now, but we should really start considering how data is serialized. (Are they going to want to do analysis on "number0" in the CSV file? Or are they going to want a computed program that adds up numeric integer values for each field?)